src/main/java/org/infobourg/betterrss/models/RssFlux.kt

```diff
@@ -0,0 +1,19 @@
+package org.infobourg.betterrss.models
+
+import org.springframework.data.annotation.Id
+import org.springframework.data.mongodb.core.mapping.Document
+import java.time.LocalDateTime
+
+//@Document
+@Document(collection = "RssFlux")
+data class RssFlux(
+        @Id
+        var id: String="",
+        var idWorkspace: String="",
+        var idChannel: String="",
+        var idMessage: String="",
+        var idThread: String="",
+        var link: String="",
+        var lastUpdate: LocalDateTime= LocalDateTime.now()
```

Comment:
It's preferable to set immutable properties (`val`) for `data class` and uses `copy` to change properties.

Reply:
I think JPA uses mutation for creating their objects. putting it as a `var` is not for us but more for the framework. But if it's not the cas, we should indeed put it as a `val`

Reply:
I changed locally to `val` and it's still working.